pulsar-transaction-contrib/pom.xml

```diff
@@ -21,8 +21,51 @@
         <artifactId>pulsar-java-contrib</artifactId>
         <version>1.0.0-SNAPSHOT</version>
     </parent>
-    <inceptionYear>2024</inceptionYear>
 
     <artifactId>pulsar-transaction-contrib</artifactId>
 
+    <dependencies>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.13.1</version>
```

Comment:
These dependency versions could be unified into properties and placed in the top-level pom.